README.md

```diff
@@ -1,2 +1,143 @@
+- [📝 License](#license)
+
+
+# 📖 Enumerable <a name="about-project"></a>
+
+In this project, we learned how to use a `module` inside our `class`. For this, we created a class called `MyList` and a module named `MyEnumerable`. The module MyEnumerable implemented a subset of the functionality of Enumerable.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/">Ruby</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Enumerable**
+- **Module**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+ <ul>
+    <li>Install a text editor like Visual Studio Codes</li>
+    <li>Make sure you have ruby installed as well in your system</li>
+ </ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/LDouglasOT/ruby_enumerable.git
+```
+
+### Install
+
+This project is considered installed once you clone it
+
+
+### Usage
+
+Once the project is cloned in your desired directory you can use it
+
+### Run tests
+
+To ensure that the ruby files pass linters run the following command
+
+```sh
+  rubocop .  
+```
+
+### Deployment
+
+You can only run this project on your local machine
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Chongtham Ruby Devi**
+
+- GitHub: [@rubydevi](https://github.com/rubydevi)
+- Twitter: [@arieabyss](https://twitter.com/ariesabyss)
+- LinkedIn: [Chongtham Ruby Devi](https://www.linkedin.com/in/chongtham-bhoomika/)
+
+👤 **Douglas Luzinda**
+
+- GitHub: [@LdouglasOT](https://github.com/LdouglasOT)
+- Twitter: [@LdouglasOT](https://twitter.com/LdouglasOT)
+- LinkedIn: [LdouglasOT](https://www.linkedin.com/in/LdouglasOT/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Improve the code to be better and better
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project share it with others
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for providing the problem for us to solve with ruby
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] @Team great work on documenting your project, however, the MIT license link doesn't work, this is because you don't have a license file in this branch, kindly consider adding it.